added missing steps to create an administrative user (#2864)
skyportal/skyportal: doc/setup.md

### Creating an administrative user

By default, no user has permission to perform system administration.

> You can give such permissions as well as other permissions to an existing user by running the following from your terminal in the `skyportal` directory:

Reviewer: ```suggestion
You can give this and other permissions to an existing user by running the following from your terminal in the `skyportal` directory:
```

```

> PYTHONPATH=. python tools/elevate_user.py --username="<username>" --role="<role>"

Reviewer: Can we change the name of the script to something a bit more descriptive, like `set_user_role`?  I think `elevate` always means up!

```

> The `username` argument is the name of the user that we want to set to a role, and the `role` argument is the role to give them. The role is optionnal, and defaults to `Super admin`.

Reviewer: ```suggestion
The role is optional, and defaults to `Super admin`.
```

Author: You want me to replace all of it by just the explanations about the role ? (i don't explain the username argument anymore ?). I made a modification that made more sense to me while you confirm this or not.

> Also, you can omit those arguments and use:

Reviewer: ```suggestion
You may also list all users and their roles:
```

```
PYTHONPATH=. python tools/elevate_user.py --list
```

> to list all users, their roles and the roles that they can be elevated to.

Reviewer: ```suggestion
to list all users, their roles and the roles that they can be set to.
```

> to list all users, their roles and the roles that they can be set to. If no arguments are passed, it will also default to listing all users.

Reviewer: ```suggestion
```

### Creating an administrative user

By default, no user has permission to perform system administration.
You can give this and other permissions to an existing user by running the following from your terminal in the `skyportal` directory:
```
PYTHONPATH=. python tools/set_user_role.py --username="<username>" --role="<role>"
```

> The `username` argument is the name of the user that we want to set to a role. The `role` argument is optional, and defaults to `Super admin`.

Reviewer: Just because the `--user` argument is obvious.

```suggestion
The `role` argument is optional, and defaults to `Super admin`.
```
